### ISSUE TITLE
GitLab integration does not work

### ISSUE THREAD

**Issue by titan83** [requilence/integram#125]
Hello.
I have noticed that from October 2 2022 gitlab integration have not been working at all.
Symptoms:
1. The bot in the telegram receive, but does not response to any commands.
2. Requests to the https://integram.org/gitlab/**** give HTTP 200 response but messages are not delivery.
Thanks.

**Comment by PeakPerformances** [comment]
yes .since few days ago ,suddenly not working anymore , @gitlab_bot stopped send msg

**Comment by noidkn** [comment]
I confirm, also does not work from October 2, 2022.
